RELEASE-NOTES.md

```diff
@@ -0,0 +1,41 @@
+# Release Notes
```

Comment:
same ques as https://github.com/redhat-ai-dev/ai-lab-samples/pull/14/files#r1823032703

Reply:
@maysunfaisal Answered with https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#discussion_r1823052094 and https://github.com/redhat-ai-dev/ai-lab-samples/pull/14#discussion_r1823057354.